docs/_includes/status-container.html

```diff
@@ -2,9 +2,6 @@
 {% assign status = site.data.status[version.status] %}
 {% if version %}
 <p class="docs-status">
-  <span class="fd-badge{% if status.type %} fd-badge--{{ status.type }}{% endif %}" style="cursor: help;" title="{{ status.description }}">
```

Comment:
I guess we may bring the status back but since we are removing it now then what about removing the property and the current statuses